deluge/common.py

```diff
@@ -46,6 +46,9 @@
     from urlparse import urljoin  # pylint: disable=ungrouped-imports
     from urllib import pathname2url, unquote_plus  # pylint: disable=ungrouped-imports
 
+if platform.system() == 'Linux':
```

Comment:
Technically `distro` supports BSD too so perhaps just check not Windows or Darwin?